jenkins/ocp-config/deploy/jenkins-master.yml

```diff
@@ -201,27 +201,34 @@ objects:
               value: '${BITBUCKET_URL}'
           image: '${DOCKER_REGISTRY}/${ODS_NAMESPACE}/jenkins-master:${ODS_IMAGE_TAG}'
           imagePullPolicy: Always
-          livenessProbe:
+          startupProbe:
+            httpGet:
+              path: /login
+              port: 8080
+              scheme: HTTP
+            timeoutSeconds: 3
+            periodSeconds: 30
+            successThreshold: 1
             failureThreshold: 30
+          livenessProbe:
             httpGet:
               path: /login
               port: 8080
               scheme: HTTP
-            initialDelaySeconds: 120
+            timeoutSeconds: 3
             periodSeconds: 10
             successThreshold: 1
-            timeoutSeconds: 3
+            failureThreshold: 30
           name: jenkins
           readinessProbe:
-            failureThreshold: 3
             httpGet:
               path: /login
               port: 8080
               scheme: HTTP
-            initialDelaySeconds: 3
+            timeoutSeconds: 3
             periodSeconds: 10
             successThreshold: 1
-            timeoutSeconds: 3
+            failureThreshold: 30
```

Comment:
Maybe this `failureThreshold` and the one of the `livenessProbe` could be reduced now that there is a separate `startupProbe`?

Reply:
I have made a quick test and we can reduce it safelly to 15 for both probes, but i would maintain the one for the StartupProbe